packages/tables/resources/lang/es/table.php

```diff
@@ -35,7 +35,7 @@
 
         'label' => 'Navegación de paginación',
 
-        'overview' => 'Mostrando :first a :last de :total resultados',
+        'overview' => '{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de un total de :total resultados',
```

Comment:
Not sure if this string is too long for the UI?

Reply:
All right, we might leave it as the original one for simplicity sake. The original line is correct (in spelling and grammar), though.

Reply:
I think we should update it, because recently this translation was changed to read differently depending on the number of records. Maybe we should just leave out `de un total`?

The English version is `{1} Showing 1 result|[2,*] Showing :first to :last of :total results`.

Reply:
Yeah, indeed. I had originally updated it from the english version. It may seem long, but it depends on the Laravel pluralization. So, yes, let's leave it as `{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de un total de :total resultados`

Reply:
Would `Se muestran de :first a :last de :total resultados` also be correct Spanish, @pathros?

Reply:
Yes, it is also correct. Either is fine. The only difference is the verbose stuff.